### ISSUE TITLE
Summary of Lab Feedback

### ISSUE THREAD

**Issue by brendoncampbell** [UBC-MDS/DSCI532_Group204_Vino#73]
The details below summarize the lab feedback from issues #70, #71, #72 

**GENERAL**
- Titles could be more descriptive (ex: 'number of observations' could be made dynamic?) (**likely actionable for milestone 3**)
- Include units on the axis ("Price ($)") (**likely actionable for milestone 3**)
- General confusion about 'value' metric. Make it more clear what points/score/rating/ranking vs. price vs. value is. (**likely actionable for milestone 3**)
   - Use the terms points/score/ranking more consistently -> lets use rating 

**Map**
- Consider showing something other than just the number of reviews?
   - Difficult because some states/counties/regions don't have enough data to make this work well
- Consider including some additional prompts or questions to get the user thinking about how it could be used. 

**Bar Chart**
- Change default to have highest to lowest first (**likely actionable for milestone 3**)
- Change tooltip labels (**likely actionable for milestone 3**)
- Confusion about why the first bar needs to be highlighted/coloured. 
- Consider adding more description about how the 'Wine Rankings' section should be used. Perhaps with a hoverable information icon?

**Heatmap**
- No need to angle the x-axis labels here (**likely actionable for milestone 3**)
- Again, consider a hoverable information icon that explains how to use it
- Graph is slow to change, consider informing the user that it takes some time to load
- Why show value not count? 
- We don't know which bottle(s) these are referring to. In the future it would be cool if we could have a way to show the actual bottles

**Comment by katieb1** [comment]
I will not be able to do the tooltip functionality on the bar chart. Since I am using a function to make dynamic axes, it is difficult to aggregate the variables to show information effectively. This is likely not a huge issue considering the tooltip is not really adding much to the plot and wasn't in our original sketch design anyway.

**Comment by jnederlo** [comment]
I altered the `Points` in the map charts to reflect `Rating` and I refactored the code/optimized file structure in merge #81. 

Additional choropleth views based on something other than "Num Reviews" is not possible on a state by state basis. This element will be included as an additional feature in accompanying plots to compliment the map with an added interactive clickable element. This will be a major feature implementation and will proceed with a later version. 

Prompts for use on maps is included in the short description on the "Geographic Analysis" tab.
